### ISSUE TITLE
set-splash ignores app id

### ISSUE THREAD

**Issue by rares-lupascu** [bamlab/react-native-make#10]
the build ignores the app id which is something like io.app.client
it uses the app segment only

**Comment by rares-lupascu** [comment]
```javascript
{
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: './android/app/src/main/java/com/app/MainActivity.java'
}
```
should be
```javascript
{
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: './android/app/src/main/java/com/io.app.client/MainActivity.java'
}
```

**Comment by rares-lupascu** [comment]
anyone?

**Comment by peteroid** [comment]
I had the same issue.

The issue is due to https://github.com/bamlab/react-native-make/blob/master/src/modules/setSplashScreen/android/service.ts#L55 where the tool assumes the android source path is same as the package name. 

You can hardcode your android source path in `./node_modules/@bam.tech/react-native-make/dist/modules/setSplashScreen/android/service.js` and it should work.

**Comment by yleflour** [comment]
Hello, currently the plugin doesn't use the `config` argument provided by `react-native-cli`I will be working on switching to it. But I will have to check that it remains backward compatible (see #6 )

**Comment by MingyuJeon** [comment]
> anyone?

You can make `io.app.client` folder and behind the folder, copy `MainActivity.java` file
After you generate splash images, remove it.
I resolve this issue in this way as a simple temporary solution.

**Comment by yleflour** [comment]
Should be fixed by #25
